api/office_structure/models.py

```diff
@@ -38,3 +38,11 @@ def add_branch(self, nodes):
         db_session.add_all(nodes)
 
         db_session.commit()
+
+    @property
+    def get_nodes(self):
```

Comment:
Good work @RachaelNantale, I think this property/field is better referred to as `nodes`.

Reply:
Thanks @ThaDeveloper, as advised by @Bonifase,  a function's name should begin with a verb

Reply:
Okay,my concern was this ```structure{
            name
            getNodes{
              id
              name
              level
            }```. Whether we could have getNodes as `nodes`

Reply:
Yes, this gets its name from the `get_nodes` function. Whichever name I use here is what I have to call the field.